srcpkgs/python3-pynest2d/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-pynest2d'
+pkgname=python3-pynest2d
+version=4.8.0
+revision=1
+wrksrc="pynest2d-${version}"
+build_style=cmake
+configure_args="-DCMAKE_CXX_STANDARD=14"
+makedepends="python3-devel libnest2d-devel python3-sip-devel pkg-config clipper-devel nlopt-devel boost-devel"
+depends="libnest2d"
```

Comment:
I think you might be missing `python3` from `hostmakedepends`?

Also, @sgn, could this benefit from sip build style?

Reply:
I think yes, I'll look into this one after finishing #27041

Reply:
Actually, no. This package uses sip4 not sip-build